api/lib/mno_enterprise/concerns/controllers/pages_controller.rb

```diff
@@ -59,9 +59,16 @@ def app_logout
   def terms
     @meta[:title] = 'Terms of Use'
     @meta[:description] = 'Terms of Use'
-    ts = MnoEnterprise::App.order_by("updated_at.desc").first.updated_at
-    @apps = Rails.cache.fetch(['pages/terms/app-list', ts]) do
-      MnoEnterprise::App.order_by("name.ac").reject{|i| i.terms_url.blank?}
+
+    ts = MnoEnterprise::App.order_by("updated_at.desc").first
```

Comment:
It's supposed to work:
```ruby
2.3.3 :001 > [].first.try(:updated_at)
 => nil 
```
http://api.rubyonrails.org/classes/Object.html#method-i-try

The safe navigation operator is a good idea but only works with ruby 2.3+ so we'd break the compatibility with older versions (as you can see with the failing specs)